### ISSUE TITLE
Exactly-once should also work without topic name included in the path 

### ISSUE THREAD

**Issue by hariprasad-k** [confluentinc/kafka-connect-hdfs#515]
This change is necessary to retain EOS semantics to support flexible storage partitioning scheme as proposed here: https://github.com/confluentinc/kafka-connect-storage-common/issues/126

**Comment by JozoVilcek** [comment]
@hariprasad-k this looks like something I am looking for too. What is holding the progress here? Did you manage to get this done somehow else?

**Comment by hariprasad-k** [comment]
> @hariprasad-k this looks like something I am looking for too. What is holding the progress here? Did you manage to get this done somehow else?

@JozoVilcek  Yes, if you apply the below patches, it does work for us to exclude topic names with EoS semantics. We are still awaiting for committers to review this change proposal though, so that we don’t need to maintain this internal fork.

https://github.com/confluentinc/kafka-connect-storage-common/pull/127/files
https://github.com/confluentinc/kafka-connect-hdfs/pull/528/files
